docs/pages/setup/reference/cli.mdx

```diff
@@ -1305,6 +1305,360 @@ $ tctl auth rotate --type=user --grace-period=200h
 $ tctl auth rotate --type=host --grace-period=8h
 ```
 
+### tctl sso test
+
+Perform end-to-end test of SSO flow using provided auth connector definition.
+
+The command supports all auth connector types: `github`, `oidc`, `saml`. The latter two require Teleport Enterprise.
+
+The testing consists of running a single end-to-end authentication request against provided auth connector definition.
+Once the request is finished, the results will be printed to console along with context-specific diagnostic informations.
+The test process will modify the list of configured auth connectors or result in certificates being issued.
+
+```code
+$ tctl [<global-flags>] sso test [<filename>]
+```
+
+<Admonition
+  type="note"
+  title="Important"
+>
+  To use this command, you must have access to `github_request`, `oidc_request`, `saml_request` resources (depending on the type of connector being tested).
+
+  If you receive "permission denied" error, modify one of your roles to allow access to:
+
+```yaml
+   - resources:
+      - github_request
+      verbs:
+      - list
+      - create
+      - read
+      - update
+      - delete
+    - resources:
+      - oidc_request
+      verbs:
+      - list
+      - create
+      - read
+      - update
+      - delete
+    - resources:
+      - saml_request
+      verbs:
+      - list
+      - create
+      - read
+      - update
+      - delete
+```
+</Admonition>
+
+
+#### Arguments
+
+- `[<filename>]` Connector resource definition file. Optional. Empty for stdin.
+
+#### Flags
+
+This command defines no flags.
+
+#### Global flags
+
+These flags are available for all commands `--debug, --config`. Run
+`tctl help <subcommand>` or see the [Global Flags section](#tctl-global-flags).
+
+#### Examples
+
+Test the auth connector from `connector.yaml`:
+
+```code
+$ tctl sso test connector.yaml
+```
+
+The command is designed to be used in conjunction with `tctl sso configure` family of commands:
+
+```code
+$ tctl sso configure ... | tctl sso test
+```
+
+The pipeline may also utilise `tee` to capture the connector generated with `tctl sso configure`. This will save the connector in `connector.yaml`:
```

Comment:
```suggestion
The pipeline may also utilize `tee` to capture the connector generated with `tctl sso configure`. This will save the connector in `connector.yaml`:
```

We tend to use the American English spelling throughout the docs, so let's be consistent here.

Reply:
Ah, I wonder if I can configure the spell checker to catch these. Thanks.